docker/single-process/environment.yml

```diff
@@ -1,7 +1,13 @@
-huginn_base:
-  environment:
-    DATABASE_ADAPTER: mysql2
-    DATABASE_NAME: huginn
-    DATABASE_USERNAME: huginn
-    DATABASE_PASSWORD: myhuginnpassword
-    APP_SECRET_TOKEN: 3bd139f9186b31a85336bb89cd1a1337078921134b2f48e022fd09c234d764d3e19b018b2ab789c6e0e04a1ac9e3365116368049660234c2038dc9990513d49c
+# This needs at least compose 1.6.0
+version: '2'
+
+services:
+  huginn_base:
+    environment:
+      DATABASE_ADAPTER: mysql2
+      DATABASE_NAME: huginn
+      DATABASE_USERNAME: huginn
+      DATABASE_PASSWORD: myhuginnpassword
+      APP_SECRET_TOKEN: 3bd139f9186b31a85336bb89cd1a1337078921134b2f48e022fd09c234d764d3e19b018b2ab789c6e0e04a1ac9e3365116368049660234c2038dc9990513d49c
+      MYSQL_PORT_3306_TCP_ADDR: mysql
```

Comment:
The mysql specific environment variables can not be in here, the break the container setup of `postgresql.yml`.